Tests/Tests/SDImageCacheTests.m

```diff
@@ -46,11 +46,15 @@ - (void)test04ClearDiskCache{
     [self.sharedImageCache clearDiskOnCompletion:^{
         [self.sharedImageCache diskImageExistsWithKey:kImageTestKey completion:^(BOOL isInCache) {
             if (!isInCache) {
-                [expectation fulfill];
+                [self.sharedImageCache calculateSizeWithCompletionBlock:^(NSUInteger fileCount, NSUInteger totalSize) {
+                    expect(fileCount).to.beLessThan(100);
```

Comment:
actually, this should be check against 0, because after clearDisk we should have 0 files in the cache.